client/js/widgets/deck.js

```diff
@@ -21,6 +21,8 @@ class Deck extends Widget {
   }
 
   cardPropertyGet(cardType, property) {
+    if(this.cards[cardId].deck.state.faceTemplates[activeFace || 0][property] !== undefined)
+      return this.cards[cardId].deck.state.faceTemplates[activeFace || 0][property];
     if(this.p('cardTypes')[cardType][property] !== undefined)
```

Comment:
I think I would expect a property in `cardTypes` to override a value in `faceTemplates`. Hard to tell.

Reply:
There is no way to assign a property to a specific face in `cardTypes`

Reply:
In order to allow a card's back side to disable `enlarge`, the faceTemplate properties need to override the cardType properties.